pallets/loans/src/market_mock.rs

```diff
@@ -0,0 +1,18 @@
+use primitives::{Rate, Ratio};
+
+use crate::{InterestRateModel, JumpModel, Market, MarketState};
+use sp_runtime::FixedPointNumber;
+
+pub const MARKET_MOCK: Market = Market {
```

Comment:
I don't think it necessary to create a new file for `MARKET_MOCK`. 
You can put it in mock.rs and can be used for benchmarking. 
https://github.com/parallel-finance/parallel/pull/259/files#diff-cc056c251f57cb266ce8797cad1318bfeb6b4912ee244838898732697c3fa9adR64

Reply:
Done